tracer/src/Datadog.Trace/Datadog.Trace.csproj

```diff
@@ -11,6 +11,8 @@
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
     <PackageReadmeFile>README.md</PackageReadmeFile>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <!-- remove LangVersion=preview when 7.0 is supported within all IDEs  -->
+    <LangVersion>preview</LangVersion>
```

Comment:
I still don't think we need/should have this 😛

Reply:
build in my IDE is still broken without it, but I guess I should try and find out why that is, removed in 24e723c8e